plugins/modules/azure_rm_openshiftmanagedcluster_info.py

```diff
@@ -0,0 +1,382 @@
+
+
+class AzureRMOpenShiftManagedClustersInfo(AzureRMModuleBaseExt):
+    def __init__(self):
+        self.module_arg_spec = dict(
+            resource_group=dict(
+                type='str'
+            ),
+            name=dict(
+                type='str'
+            )
+        )
+
+        self.resource_group = None
+        self.name = None
+
+        self.results = dict(changed=False)
+        self.mgmt_client = None
+        self.state = None
+        self.url = None
+        self.status_code = [200]
+
+        self.query_parameters = {}
+        self.query_parameters['api-version'] = '2020-04-30'
+        self.header_parameters = {}
+        self.header_parameters['Content-Type'] = 'application/json; charset=utf-8'
+
+        self.mgmt_client = None
+        super(AzureRMOpenShiftManagedClustersInfo, self).__init__(self.module_arg_spec, supports_check_mode=True, supports_tags=False)
+
+    def exec_module(self, **kwargs):
+
+        for key in self.module_arg_spec:
+            setattr(self, key, kwargs[key])
+
+        self.mgmt_client = self.get_mgmt_svc_client(GenericRestClient,
+                                                    base_url=self._cloud_environment.endpoints.resource_manager)
+
+        if (self.resource_group is not None and self.name is not None):
+            self.results['clusters'] = self.get()
+        elif (self.resource_group is not None):
+            self.results['clusters'] = self.list()
+        else:
+            self.results['clusters'] = self.listall()
+        return self.results
+
+    def get(self):
+        response = None
+        results = {}
+        # prepare url
+        self.url = ('/subscriptions' +
+                    '/{{ subscription_id }}' +
+                    '/resourceGroups' +
+                    '/{{ resource_group }}' +
+                    '/providers' +
+                    '/Microsoft.RedHatOpenShift' +
+                    '/openShiftClusters' +
+                    '/{{ open_shift_managed_cluster_name }}')
+        self.url = self.url.replace('{{ subscription_id }}', self.subscription_id)
+        self.url = self.url.replace('{{ resource_group }}', self.resource_group)
+        self.url = self.url.replace('{{ open_shift_managed_cluster_name }}', self.name)
+
+        try:
+            response = self.mgmt_client.query(self.url,
+                                              'GET',
+                                              self.query_parameters,
+                                              self.header_parameters,
+                                              None,
+                                              self.status_code,
+                                              600,
+                                              30)
+            results = json.loads(response.text)
+            # self.log('Response : {0}'.format(response))
+        except CloudError as e:
+            self.log('Could not get info for @(Model.ModuleOperationNameUpper).')
```

Comment:
If the above exception, no get information, no value can be obtained.
```suggestion
            self.log('Could not get info for @(Model.ModuleOperationNameUpper).')
            return {}
```